src/components/LoginButton/test.spec.js

```diff
@@ -43,11 +43,8 @@ it('Login button has the right URL after cancelling log in', () => {
     );
   });
 
-  const expectedUrl = new URL(option.url);
-  expectedUrl.searchParams.set('next', 'https://open-forms.nl/digid-form/?_start=1');
-
   const actualUrl = container.getElementsByClassName('openforms-button')[0].href;
 
-  expect(actualUrl).toEqual(expectedUrl.toString());
+  expect(actualUrl).toEqual(option.url);
```

Comment:
doesn't this break the whole redirect flow if the `next` param is not in the URL anymore?

Reply:
It is not the `LoginButton` now that makes the URL but the 'smart' `LoginOptions` component. So the `LoginButton` just uses the URL that is passed as prop.

Maybe this test makes no sense anymore and I should test the `LoginOptions` component.